app/Bookstore.Cdk/EC2ComputeStack.cs

```diff
@@ -1,4 +1,5 @@
 using Amazon.CDK;
+using Amazon.CDK;
```

Comment:
It looks like Amazon.CDK might have been imported twice.

Reply:
Removed the duplicate.